src/metrics.cpp

```diff
@@ -204,20 +193,20 @@ int printStats(bool mining)
     int lines = 4;
 
     int height;
-    int64_t tipmediantime;
+    int64_t time;
     size_t connections;
     int64_t netsolps;
     {
         LOCK2(cs_main, cs_vNodes);
         height = chainActive.Height();
-        tipmediantime = chainActive.Tip()->GetMedianTimePast();
+        time = chainActive.Tip()->GetBlockTime();
         connections = vNodes.size();
         netsolps = GetNetworkHashPS(120, -1);
     }
     auto localsolps = GetLocalSolPS();
 
     if (IsInitialBlockDownload(Params())) {
-        int netheight = EstimateNetHeight(height, tipmediantime, Params());
+        int netheight = EstimateNetHeight(Params().GetConsensus(), height, time);
```

Comment:
Here's where `pindexBestHeader` (or `pindexBestHeader->nHeight, pindexBestHeader->nTime`) would need to be passed in.